README.md

```diff
@@ -0,0 +1,70 @@
+<div align="center">
+  <img src="https://github.com/hmartiins/chat/assets/51277667/fca09cd1-d66a-4843-a6f4-cbbc1bfd738f" width="200" />
+
+  <h1>Gotter Chat</h1>
+  <p>💬 Application for exchanging messages in real time 📱</p>
+</div>
+
+## Project Overview
+
+This project is a backend implementation in Go for a real-time chat system using Socket.IO. The primary goal is to create a scalable and efficient server that facilitates communication between multiple clients in real-time.
+
+### Features
+
+- **Real-time Communication**: Utilizing Socket.IO for bidirectional communication between the server and clients, enabling instant message updates.
+- **Scalability**: Designed to handle a large number of concurrent connections efficiently, ensuring smooth communication even with a high volume of users.
+- **User Authentication**: Implement a secure authentication mechanism to validate users and manage access to chat features.
+- **Message Persistence**: Optionally, store chat messages in a database to provide message history and continuity across user sessions.
+- **Error Handling**: Implement robust error handling to gracefully manage unexpected scenarios, ensuring the stability of the chat server.
+- **Logging**: Utilize logging to keep track of important events and errors for monitoring and debugging purposes.
+
+## Getting Started
+
+### Prerequisites
+
+- [Go](https://golang.org/) installed on your machine.
+- [Socket.IO Go Library](https://github.com/googollee/go-socket.io) installed.
+
+### Installation
+
+1. Clone the repository:
+
+    ```bash
+    git clone https://github.com/your-username/chat-backend-go
+    ```
+
+2. Navigate to the project directory:
+
+    ```bash
+    cd chat
+    ```
+
```

Comment:
add the .env configuration session before running the project. Using "cp .env.example .env"